modules/tools/mapshow/libs/map.py

```diff
@@ -147,6 +147,36 @@ def _draw_lane_details(self, lane, ax, color_val):
             arrowprops=dict(arrowstyle='-|>', connectionstyle='arc3,rad=-0.2',
                             fc=color_val, ec=color_val, alpha=0.5))
 
+    def draw_pnc_junctions(self, ax):
+        cnt = 1
```

Comment:
cnt = 1 + len(self.map_pb.pnc_junction) ?

Reply:
cnt is a counter, incrementing, used to decide/differentiate label color